src/components/Navbar/SettingsDrawer/QuranFontSection.tsx

```diff
@@ -108,6 +111,11 @@ const QuranFontSection = () => {
   return (
     <Section>
       <Section.Title>{t('fonts.quran-font')}</Section.Title>
+      <Section.Row>
+        <div className={styles.versePreviewContainer}>
+          <VersePreview />
```

Comment:
Tajweed preview direction is flipped on mobile for LTR languages
<img width="269" alt="Screen Shot 2021-11-25 at 16 07 47" src="https://user-images.githubusercontent.com/15169499/143412514-1184d33e-4bd2-4b6a-a770-b3676d283ec2.png">